gensim/models/sent2vec.py

```diff
@@ -75,6 +75,25 @@ class ModelDictionary():
     """
 
     def __init__(self, t, bucket, minn, maxn, max_vocab_size=30000000, max_line_size=1024):
+        """
+        Initialize a sent2vec dictionary.
```

Comment:
Numpy-style please: https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt and http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html

Reply:
Sorry, my bad! I used the word2vec code as a reference. I've updated the docstrings. Kindly verify in the latest commit.